srv/components/provisioner/scripts/install.sh

```diff
@@ -45,6 +46,7 @@ _usage()
     echo "
 Options:
   -t|--target-build BUILD_URL        Target Cortx build to deploy
+  -o|--opensource                          Skip looking for OS ISO image in opensource environment
```

Comment:
We can get rid of this parameter and accept the path with -t option like this: "-t file://var/artifacts/0"
So, if the tartget_build location has only one ISO (and that too cortx_single.iso), we can assume that it's for community build and ignore the ISO for OS.